xml/System.Reflection.Metadata/Document.xml

```diff
@@ -23,8 +23,16 @@
     </Attribute>
   </Attributes>
   <Docs>
-    <summary>To be added.</summary>
-    <remarks>To be added.</remarks>
+    <summary>The source document in the debug metadata.</summary>
+    <remarks>
+      <format type="text/markdown"><![CDATA[
+
+## Remarks
+
+See also https://github.com/dotnet/corefx/blob/master/src/System.Reflection.Metadata/specs/PortablePdb-Metadata.md#document-table-0x30.
```

Comment:
Some markdown parsers will not encode inline https as a link. It's better to specify it explicitly. I'm also editing the text here to make it more user friendly.

```suggestion
For more information, see [Portable PDB v1.0: Format Specification](https://github.com/dotnet/corefx/blob/master/src/System.Reflection.Metadata/specs/PortablePdb-Metadata.md#document-table-0x30).
```

Reply:
Thank you very much @Thraka for taking a look. I'll apply your suggestion.